modules/cnv-troubleshooting-vmware-vm-import.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+// * cnv/cnv_virtual_machines/cnv_importing_vms/cnv-importing-vmware-vm.adoc
+[id='troubleshooting-vmware-vm-import-{context}']
+= Troubleshooting a VMware virtual machine import
+
+If an imported virtual machine's status is `Import error: (VMware)`, you can check the Conversion Pod log for errors:
+
+. Obtain the Conversion Pod name:
++
+----
+$ oc get pods -n <project> | grep v2v <1>
+kubevirt-v2v-conversion-f66f7d-zqkz7            1/1     Running     0          4h49m
+----
+<1> Specify the project of your imported virtual machine.
+
+. Obtain the Conversion Pod log:
++
+----
+$ oc logs kubevirt-v2v-conversion-f66f7d-zqkz7 -f -n <project>
+----
+
+= Error messages
+
+* Virtual machine event error message: `Readiness probe failed`
++
+Log message: `INFO - have error: ('virt-v2v error: internal error: invalid argument: libvirt domain ‘v2v_migration_vm_1’ is running or paused. It must be shut down in order to perform virt-v2v conversion',)"`
++
+Resolution: Shut down the virtual machine before importing it.
+
+ifeval::[{CNVVersion} == "2.3"]
```

Comment:
Should this be showing for 4.4/4.5? Because I can see that the CNVVersion is set to 2.3 in the cnv-document-attributes.adoc file in master, but this section doesn't display in the preview build.

Reply:
Strange. This worked when I created the preview build. But if the topic_map is commented out, I imagine it won't appear at all.

Reply:
> Should this be showing for 4.4/4.5? Because I can see that the CNVVersion is set to 2.3 in the cnv-document-attributes.adoc file in master, but this section doesn't display in the preview build.

That depends on whether CNV 2.3 is going to be visible for 4.4/4.5 or not. As long as CNV version is 2.3, that section should appear.

Reply:
So the known issues section is still not showing for me when I do a build. I checked other places where we do an ifeval, and they all put quotes around the variable.

```
ifeval::["{CNVVersion}" == "2.3"] 
```

I tried this and then this section shows for me.